src/field.h

```diff
@@ -136,7 +137,11 @@ static void secp256k1_fe_clear(secp256k1_fe *a);
  */
 static int secp256k1_fe_is_zero(const secp256k1_fe *a);
 
-/** Check the "oddness" of a field element. Requires the input to be normalized. */
+/** Determine whether a (mod p) is odd.
+ *
+ * On input a must be a valid normalized field element.
```

Comment:
nit: "On input, a" makes this much easier to parse for my brain. Maybe it's a good idea add commas after every "On input"/"On output"